stage0/run.go

```diff
@@ -47,21 +43,17 @@ import (
 	"github.com/coreos/rocket/pkg/lock"
 	ptar "github.com/coreos/rocket/pkg/tar"
 	"github.com/coreos/rocket/version"
-
-	"github.com/coreos/rocket/stage0/stage1_init"
-	"github.com/coreos/rocket/stage0/stage1_rootfs"
 )
 
 const (
-	initPath  = "stage1/init"
+	initPath  = "stage1/rootfs/init"
```

Comment:
I think that is a good idea (and the spec demonstrating some real world utility! :-)